modules/@ergonode/dashboard/src/components/Pages/EmptyDashboardPage.vue

```diff
@@ -250,19 +256,22 @@ export default {
 
     .dashboard-stats {
         display: flex;
+        justify-content: flex-end;
         align-items: center;
-        width: 450px;
-        height: 300px;
-        margin: 40px 120px 40px 0;
+        width: 358px;
+        height: 358px;
+        margin: 40px 170px 40px 0;
         background:
-            url("~@Core/assets/images/login/login_password_recovery_face.png")
+            url("~@Dashboard/assets/images/dashboard_man.svg")
             no-repeat
             right
             bottom;
         background-size: contain;
 
-        &__image {
-            height: 280px;
+        &__product-count-chart {
+            width: 174px;
+            height: 174px;
+            margin-right: -130px;
```

Comment:
What's the reason about using negative margin? It might produce unexpected results on low ration screens